### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by annabasenyuk**
- [DEMO LINK](https://annabasenyuk.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://annabasenyuk.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 12px 70px;
+    border: 1px solid #f3f3f3;
```
Comment (mifyy) [inline src/styles/blocks/card.scss:68]: Creating a border for an inactive button wasn't the task requirement. If it is to be created, I think it should be the same color as the button itself (or background) when we do not hover over it.

File: src/styles/blocks/stars.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+  }
+
+  &__star--0 :nth-child(-n + 4) {
```
Comment (mifyy) [inline src/styles/blocks/stars.scss:17]: I don't this that increasing specificity is a must here, :nth-child(-n + 4) will work.

**Review by mifyy — CHANGES_REQUESTED** [review]
1 minor issue with following the mockup.

File: src/index.html [hunk 3]
```diff
@@ -15,48 +15,38 @@
...
+      >
+      <h1 class="card__header">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="card__code">Product code: 195434</p>
 
-          <div class="price">
-            <span class="price__article">Price:</span>
-            <span class="price__price">$2,199</span>
-          </div>
+      <div class="card__rating">
+        <div class="stars stars__star--0">
```
Comment (tetyana-pol) [inline src/index.html:28]: should be class stars--0

**Review by tetyana-pol — APPROVED** [review]

File: src/index.html [hunk 4]
```diff
@@ -5,9 +5,48 @@
...
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card card--width" data-qa="card">
+      <img
```
Comment (superpooperxxx) [inline src/index.html:19]: `<article>`😍

File: src/index.html [hunk 5]
```diff
@@ -5,9 +5,48 @@
...
+        class="card__photo"
+        src="./images/imac.jpeg"
+        alt="Imac Monoblock photo"
+      >
+      <h1 class="card__header">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="card__code">Product code: 195434</p>
+
```
Comment (superpooperxxx) [inline src/index.html:26]: 1. Fix everywhere
image
2. You can have only 1 `h1` per page, but we can have a lot of cards. I think `h3` fits better here.

File: src/index.html [hunk 6]
```diff
@@ -5,9 +5,48 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+         Reviews: 5
+      </div>
+      <p class="card__price">
+        <span class="card__title">Price:</span>
+        <span class="card__sum">$2,199</span>
+      </p>
```
Comment (superpooperxxx) [inline src/index.html:41]: `.card__title` is definitely not 'title'. I suggest `.card__price-text` and `.price__price-value`.
Good job on using `<p>` and `<span>`🔥

File: src/styles/blocks/card.scss [hunk 7]
```diff
@@ -0,0 +1,84 @@
+$grey-color: #616070;
+$blue-color: #00acdc;
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
```
Comment (superpooperxxx) [inline src/styles/blocks/card.scss:6]: Always set `box-sizing: border-box` as
```
* {
  box-sizing: border-box;
}
```

File: src/styles/blocks/card.scss [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 16px;
+  padding-top: 32px;
+
+  &--width {
+    width: 200px;
+  }
```
Comment (superpooperxxx) [inline src/styles/blocks/card.scss:14]: I don't think, that you need this modifier.

File: src/styles/normalize.scss [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+}
+
+body {
+  margin: 0;
+}
```
Comment (superpooperxxx) [inline src/styles/normalize.scss:12]: This needs to go to `reset.scss`

File: src/styles/blocks/card.scss [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__header {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
```
Comment (superpooperxxx) [inline src/styles/blocks/card.scss:28]: I suggest always reseting default margins

File: src/index.html (hunk 3 above)
Comment (superpooperxxx) [reply src/index.html:28]: Fix the comment of your mate

File: src/styles/blocks/stars.scss (hunk 2 above)
Comment (superpooperxxx) [reply src/styles/blocks/stars.scss:17]: With this code you are ready only for the case when the rating is 4 stars. I think you need to generate modifiers using scss loop. (If you need help, tag me)

**Review by superpooperxxx — CHANGES_REQUESTED** [review]
You did an awesome job🔥Most of the comments are recommendations, but better fix them all😉

**Review by superpooperxxx — APPROVED** [review]
🔥🔥🔥

**Pull request closed without merging**